internal/character/qingque/skill.go

```diff
@@ -0,0 +1,59 @@
+package qingque
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Skill key.Modifier = "qingque-skill"
+)
+
+type skillState struct {
+	damageBoost float64
+}
+
+func init() {
+	modifier.Register(Skill, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
+		Stacking:   modifier.ReplaceBySource,
+		MaxCount:   4,
+		Listeners: modifier.Listeners{
+			OnAdd:    skillOnAdd,
+			OnPhase2: skillOnPhase2,
+		},
+		CountAddWhenStack: 1,
+	})
+}
+func (c *char) Skill(target key.TargetID, state info.ActionState) {
+	extraDamage := 0.0
+	if c.info.Traces["1201102"] {
+		extraDamage = 0.1
+	}
+	c.engine.AddModifier(target, info.Modifier{
+		Name:   Skill,
+		Source: c.id,
+		State: skillState{
+			damageBoost: extraDamage + skill[c.info.SkillLevelIndex()],
+		},
+	})
+	c.drawTile()
+	c.drawTile()
```

Comment:
If she draws 4 tiles of the same face by her skill, shouldn't her skill give her the "hidden hand" state?